exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,15 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount;// finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let total = studentCount + mentorCount;
+let enoughSpaceInRoom = total <= 25;  // finish this statement
```

Comment:
This works, but since you are storing the room capacity above it's better to do this:
```
let enoughSpaceInRoom = total <= roomMaxCapacity;
```